castervoice/rules/ccr/rust_rules/rust.py

```diff
@@ -107,6 +107,8 @@ class Rust(MergeRule):
             R(Text("'")),
         "static":
             R(Text("static ")),
+        "self":
+            R(Text("self")),
```

Comment:
`self` is a frequently used command and I thought it would be useful to have it here. I do not have a space after because it is often used in a cascading context where `self dot` is useful